### PR TITLE
Remove all superscript asterisks

### LINKED ISSUE #18: Double superscript
The asterisk plus prime is very hard to read.

`* Let ''R<sup>*</sup>' = R<sup>*</sup><sub>1</sub> + b⋅R<sup>*</sup><sub>2</sub>''`

https://github.com/jonasnick/bips/blame/aab260c37112b2010266fd8f0d3d56ea1111a2cc/bip-musig2.mediawiki#L382

Sorry for opening an entire issue for this but I don't have a great suggestion right now

### PR DESCRIPTION
**Pull request by real-or-random**
I believe their purpose to stress that the variable belongs to a single signer.
But they're not necessary to indicate that  `pk` refers to an individual public
key because we use `aggpk` for the aggregate public key.
In case of `R`, this commit replaces the asterisks by `Rp` (for _p_artial).

Fixes #18.

Alternative to #46 . When I had this fully ready, I realized that #47 may be much simpler, so we have a choice now.

### CODE REVIEW

File: bip-musig2.mediawiki [hunk 1]
```diff
@@ -335,8 +335,8 @@ Input:
 ** Let ''extra_in = empty_bytestring''
 * Let ''k<sub>i</sub> = int(hash<sub>MuSig/nonce</sub>(rand || bytes(1, len(aggpk)) || aggpk || m_prefixed || bytes(4, len(extra_in)) || extra_in || bytes(1, i - 1))) mod n'' for ''i = 1,2''
 * Fail if ''k<sub>1</sub> = 0'' or ''k<sub>2</sub> = 0''
-* Let ''R<sup>*</sup><sub>1</sub> = k<sub>1</sub>⋅G, R<sup>*</sup><sub>2</sub> = k<sub>2</sub>⋅G''
-* Let ''pubnonce = cbytes(R<sup>*</sup><sub>1</sub>) || cbytes(R<sup>*</sup><sub>2</sub>)''
+* Let ''R<sub>1</sub> = k<sub>1</sub>⋅G, R<sub>2</sub> = k<sub>2</sub>⋅G''
+* Let ''pubnonce = cbytes(R<sub>1</sub>) || cbytes(R<sub>2</sub>)''
```
Comment (jonasnick) [inline bip-musig2.mediawiki:339]: R -> Rp for consistency with the values in partial_sig_verify?

File: bip-musig2.mediawiki [hunk 2]
```diff
@@ -402,9 +402,9 @@ Input:
 * Fail if ''d' = 0'' or ''d' &ge; n''
 * Let ''P = d'⋅G''
 * Let ''a = GetSessionKeyAggCoeff(session_ctx, P)''; fail if that fails
-* Let ''gp = 1'' if ''has_even_y(P)'', otherwise let ''gp = -1 mod n''
+* Let ''gpk = 1'' if ''has_even_y(P)'', otherwise let ''gpk = -1 mod n''
```
Comment (jonasnick) [inline bip-musig2.mediawiki:405]: It seems that `gp` should be updated in the refernece code as well, but #37 will remove `gp` anyway.

File: bip-musig2.mediawiki [hunk 3]
```diff
@@ -430,17 +430,17 @@ Input:
...
-'''''PartialSigVerifyInternal(psig, pubnonce, pk<sup>*</sup>, session_ctx)''''':
+'''''PartialSigVerifyInternal(psig, pubnonce, pk, session_ctx)''''':
 * Let ''(Q, gacc, _, b, R, e) = GetSessionValues(session_ctx)''; fail if that fails
-* Let ''s = int(psig)''; fail if ''s &ge; n''
-* Let ''R<sup>*</sup><sub>1</sub> = cpoint(pubnonce[0:33]), R<sup>*</sup><sub>2</sub> = cpoint(pubnonce[33:66])''
-* Let ''R<sup>*</sup>' = R<sup>*</sup><sub>1</sub> + b⋅R<sup>*</sup><sub>2</sub>''
-* Let ''R<sup>*</sup> = R<sup>*</sup>' '' if ''has_even_y(R)'', otherwise let ''R<sup>*</sup> = -R<sup>*</sup>' ''
+* Let ''sp = int(psig)''; fail if ''s &ge; n''
+* Let ''Rp<sub>1</sub> = cpoint(pubnonce[0:33]), Rp<sub>2</sub> = cpoint(pubnonce[33:66])''
+* Let ''Rp' = Rp<sub>1</sub> + b⋅Rp<sub>2</sub>''
```
Comment (jonasnick) [inline bip-musig2.mediawiki:437]: Should we also rename `R -> Rp` in nonce generation? This makes clear that these nonces are (also) partial nonces.

File: bip-musig2.mediawiki [hunk 4]
```diff
@@ -430,17 +430,17 @@ Input:
 * Run ''PartialSigVerifyInternal(psig, pubnonce<sub>i</sub>, pk<sub>i</sub>, session_ctx)''
 * Return success iff no failure occurred before reaching this point.
 
-'''''PartialSigVerifyInternal(psig, pubnonce, pk<sup>*</sup>, session_ctx)''''':
+'''''PartialSigVerifyInternal(psig, pubnonce, pk, session_ctx)''''':
```
Comment (jonasnick) [inline bip-musig2.mediawiki:433]: There are still occurences of `pk<sup>*</sup>` in the remarks section.

Comment (jonasnick) [inline bip-musig2.mediawiki:433]: There are still occurences of `pk<sup>*</sup>` in the remarks section.

**Review by jonasnick — COMMENTED** [review]
note: this probably should have a changelog entry

**Comment by real-or-random** [comment]
@jonasnick would you prefer this one or #46 ?

**Comment by jonasnick** [comment]
We're going with #46

**Pull request closed without merging**